README.md

```diff
@@ -205,6 +225,33 @@ You configure ExternalDNS later in the process.
     )
     ```
 
+1. Update Network Policies
+
+    ```bash
+    ./bin/ck8s update-ips both apply
```

Comment:
Could we first run the `dry-run` so we see if there are any IP's or ports being removed that are not supposed to be removed?
I am thinking of the many cases where the `update-ips` script removed object storage ports or CIDR subnets that affected the environment afterwards.